pegasus/test/test_v2_user_routes.rb

```diff
@@ -166,18 +165,22 @@ def expected_v2_students_id_hash_for(user)
 
       it 'returns 403 "Forbidden" when signed in as another student' do
         with_role FakeDashboard::SELF_STUDENT
-        @pegasus.post "/v2/students/#{FakeDashboard::STUDENT[:id]}/update",
-          {name: NEW_NAME}.to_json,
-          'CONTENT_TYPE' => 'application/json;charset=utf-8'
-        assert_equal 403, @pegasus.last_response.status
+        Dashboard.db.transaction(rollback: :always) do
```

Comment:
You can get this for free by deriving from [SequelTestCase](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/test/sequel_test_case.rb)